addons/sale_mrp/models/stock_rule.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class StockRule(models.Model):
+    _inherit = 'stock.rule'
+
+    def _prepare_mo_vals(self, product_id, product_qty, product_uom, location_dest_id, name, origin, company_id, values, bom):
```

Comment:
Why not fixing in `_prepare_procurement_values` ?
So we have a `Procurement` with correct values, in case we need `product_description_variants`, somewhere else, it will already be correct

Reply:
Very good point ! I'll try that :slightly_smiling_face: